source/services/dataContracts/baseDataService/baseData.service.ts

```diff
@@ -41,16 +41,17 @@ export class BaseDataService<TDataType extends IBaseDomainObject, TSearchParams>
         return this._endpoint;
     }
 
-    private getItemEndpoint(id: number): string {
-        return this.endpoint + '/' + id.toString();
+    private getItemEndpoint(id: number, endpoint?: string): string {
```

Comment:
When is an endpoint passed in instead of using the internal one (this.endpoint)?